### DISCUSSION TITLE
I need help with deploying my Nextjs front end and Express-Graphql back end

### DISCUSSION THREAD

**Discussion by Segun98** [vercel/next.js#12253] [Help]
[https://github.com/Segun98/Blog-Nextjs](url)

There is my repo. 

Any help wpuld be appreciated

**Comment by Manubi** [comment]
your link is showing a 404.
Also a little bit more context would be helpful. If you ask your questions like that, no one is going to help you because it's just a waste of time for everyone who would like to help.

**Reply by Segun98** [reply]
Thank you. It was my first every post and I was confused at the time I made the post. I have successfully deployed my project and I have posted a (new) more detailed issue here https://github.com/zeit/next.js/discussions/12292 [](https://github.com/zeit/next.js/discussions/12292)
